scripts/scil_recognize_multi_bundles.py

```diff
@@ -63,11 +55,11 @@ def _build_arg_parser():
         description=__doc__,
         epilog=EPILOG)
 
-    p.add_argument('in_tractogram',
+    p.add_argument('in_tractograms', nargs='+',
                    help='Input tractogram filename (.trk or .tck).')
     p.add_argument('in_config_file',
                    help='Path of the config file (.json)')
-    p.add_argument('in_models_directories', nargs='+',
+    p.add_argument('in_directory',
```

Comment:
Not clear to me what "... directories contain model" -> for you, this is the /zenodo_cleaned_fss/atlas/ directory right?  which contains one or many directories containing the trk files of the bundle models. I would try to rephrase for clearer English